include/lsst/afw/geom/SpanSet.h

```diff
@@ -617,6 +620,23 @@ class SpanSet {
      */
     std::vector<std::shared_ptr<geom::SpanSet>> split() const;
 
+protected:
+    /* Returns the name used by the persistence layer to identify the SpanSet class
+     */
+    virtual std::string getPersistenceName() const;
+
+    /* Return a string corresponding to the python module that SpanSets lives in
+     */
+    virtual std::string getPythonModule() const;
```

Comment:
I assume this is standard. But I find this explicit mixing of C++ and Python API extremely ugly.